packages/server/src/index.ts

```diff
@@ -1 +1,23 @@
-export default {};
+import {Property, Signature} from "../../message/src/index";
+
+enum KeyType {
+  EDDSA,
+  ECDSA,
+}
+
+type SecretKey = {
+  type: KeyType;
+  key: string;
+}
+
```

Comment:
These `enum` and `type` have not been exported because they are only used for the server.